### PR TITLE
Added changes for resdesign of welcome site

### PR DESCRIPTION
**Pull request by Pujarini**
Desktop version: 
https://user-images.githubusercontent.com/34391629/140087591-7ab43302-de33-459c-b325-bb5663e6b7a5.mov

Mobile version :

https://user-images.githubusercontent.com/34391629/140088168-b716ec18-858a-4c0e-aefa-2ec7b5bddb24.mov

### CODE REVIEW

**Review by Rucha1499 — APPROVED** [review]
image

Can we remove these files?

File: css/events/events.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  transform: scale(1.1);
+}
+
+.event-details {
+  display: block;
+  text-align: center;
+}
+
+.event-title {
+  font-size: 4em;
```
[the remark this answers is not in the document]
Comment (harshith-venkatesh) [reply css/events/events.css:26]: expecting your opinions for the same

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: css/events/events.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/events/events.css:26]: I will stick to rem then.

File: css/common/main.css [hunk 2]
```diff
@@ -163,3 +144,78 @@ footer a {
...
+
+.events {
+  color: #604af0;
+  border-bottom: 1px solid;
+  margin: 0 0.25rem;
+}
+.video-holder {
+  width: 100%;
+  background-color: #f7f4f4;
+  padding: 3rem 28rem;
```
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/common/main.css:196]: Is it more or fine?

File: css/common/main.css [hunk 3]
```diff
@@ -163,3 +144,78 @@ footer a {
...
+
+.upcoming-event {
+  font-size: 1.5rem;
+  margin-top: 2.5rem;
+  text-align: center;
+}
+
+.api-event {
+  margin: 15px 5px;
+  font-size: 20px;
```
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/common/main.css:170]: okay will change it to rem.

File: css/common/main.css [hunk 4]
```diff
@@ -13,21 +13,22 @@ body {
   font-size: 16px;
 }
 
-main {
+/* main {
```
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/common/main.css:16]: Actually this is the css for the prev changes so for safer side i didn't remove it

File: css/events/events.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/events/events.css:26]: I have added these files as it is without any change

File: css/common/main.css [hunk 5]
```diff
@@ -163,3 +144,78 @@ footer a {
...
+}
+
+.upcoming-event {
+  font-size: 1.5rem;
+  margin-top: 2.5rem;
+  text-align: center;
+}
+
+.api-event {
+  margin: 1.5rem 1.5rem;
```
Comment (swarajpure) [inline css/common/main.css:169]: Could be `margin: 1.5rem`?

File: css/common/main.css [hunk 6]
```diff
@@ -163,3 +144,78 @@ footer a {
...
+  font-size: 1.5rem;
+  border-bottom: 1px solid;
```
Comment (swarajpure) [inline css/common/main.css:171]: no color for border?

File: css/common/main.css [hunk 7]
```diff
@@ -163,3 +144,78 @@ footer a {
...
+.events-box {
+  margin-top: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-flow: row wrap;
+}
+
+.events {
+  color: #604af0;
+  border-bottom: 1px solid;
```
Comment (swarajpure) [inline css/common/main.css:190]: no color for border?

File: css/common/main.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (swarajpure) [reply css/common/main.css:196]: That would normally be 28 * 16px = 448px
Isn't that too much

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please remove the `.DS_Store` files, those seem to unnecessary
Added few comments for CSS, please check

Great job in here! 🔥

**Review by Rucha1499 — DISMISSED** [review]
Can we remove the `.DS_Store` files?

File: index.html [hunk 8]
```diff
@@ -54,71 +54,86 @@
...
+          target="_blank"
+          rel="noopener noreferrer"
+          >members.</a
+        >
+      </h3>
       <p class="info-repo">
         The contents of this website are deployed from this
         <a
-          href="https://github.com/Real-Dev-Squad/website-welcome"
+          href="https://github.com/Real-Dev-Squad/website"
```
[the remark this answers is not in the document]
Comment (Pujarini) [reply index.html:136]: okay will do that

File: css/common/main.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (Pujarini) [reply css/common/main.css:196]: yes. will resolve this

File: css/common/main.css (hunk 5 above)
Comment (Pujarini) [reply css/common/main.css:169]: yes. will do that.

File: css/common/main.css (hunk 7 above)
Comment (Pujarini) [reply css/common/main.css:190]: no, i have taken all of the anchor tags as one color.

File: css/common/main.css (hunk 6 above)
Comment (Pujarini) [reply css/common/main.css:171]: taken anchor tags as one color

File: css/events/events.css [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+@media screen and (max-width: 440px) {
+  .event-title {
+    font-size: 1.95em;
+    font-weight: 700;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  .event-title {
+    font-size: 3.3em;
```
Comment (harshith-venkatesh) [inline css/events/events.css:47]: Why are we following two different units for font-size, could you let me through any resources related to this decision

**Review by harshith-venkatesh — DISMISSED** [review]

**Comment by harshith-venkatesh** [comment]
> Please remove the `.DS_Store` files, those seem to unnecessary Added few comments for CSS, please check
> 
> Great job in here! 🔥

Indeed (y)

**Comment by Ishikabaid** [comment]
Can you please add screenshots of the updated screen @Pujarini

**Comment by Pujarini** [comment]
> Can you please add screenshots of the updated screen @Pujarini

@Ishikabaid Find screenshot here for removed underline

image

**Review by Ishikabaid — APPROVED** [review]
Good Job!🎉

**Review by swarajpure — APPROVED** [review]

**Pull request merged**